app/novu/zod/schemas.ts

```diff
@@ -0,0 +1,56 @@
+import { z } from 'zod';
+
+// read more about zod at the official webside: https://zod.dev/
+export const emailListElementTypeSchema = z.object({title: z.string().default(""), body: z.string().default("")})
+
+export const componentTypeSchema = z.discriminatedUnion('componentType', [
+  z.object({ componentType: z.literal('text'), componentText: z.string().default(""), align:z.enum(["left", "center", "right"]).default("center")}),
+  z.object({ componentType: z.literal('list'), componentText: z.string().default(""), componentListItems: z.array(emailListElementTypeSchema)}),
+  z.object({ componentType: z.literal('image'), componentText: z.string().default(""), src: z.string().default("")}),
+  z.object({ componentType: z.literal('heading'), componentText: z.string().default("")}),
+  z.object({ componentType: z.literal('button'), componentText: z.string().default("")}),
+  z.object({ componentType: z.literal('divider')}),
+  z.object({ componentType: z.literal('users')}),
+]);
+
+export const zodControlSchema = z.object({
+  components: z.array(componentTypeSchema).default([{
+    "componentType": "heading",
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Tidy up to remove unneeded quotes across the schema‏
```suggestion
    componentType: "heading",
```